src/runs.c

```diff
@@ -90,6 +90,41 @@ r_obj* vec_identify_runs(r_obj* x) {
 
 // -----------------------------------------------------------------------------
 
+r_obj* ffi_vec_run_sizes(r_obj* x) {
+  return vec_run_sizes(x);
+}
+
+r_obj* vec_run_sizes(r_obj* x) {
+  const bool start = false;
+  r_obj* where = KEEP(vec_detect_run_bounds0(x, start));
```

Comment:
Should this be called `vec_detect_run_start0()`? It'd then be immediately clear what it's doing.

Also should the bool version have `_bool` suffix? The `0` suffix doesn't convey information if you don't already know what it means in this case.

Reply:
It actually computes the _ends_ here because `start = false`. It is "bounds" because it can do either, and this algorithm needs the ends.

Reply:
It should probably be an enum `VCTRS_RUN_BOUND_starts` and `VCTRS_RUN_BOUND_ends` or something like that to make it clearer

Reply:
- switched to enum here https://github.com/r-lib/vctrs/pull/1774
- switched to `_bool` suffix here https://github.com/r-lib/vctrs/commit/864e92ec9a11863cdedde96385e1795759fe111b

Reply:
ah gotcha. Good call regarding the enum.